infra/eligibility_screener/template/ecs.tf

```diff
@@ -0,0 +1,100 @@
+# security group for screener
+resource "aws_security_group" "allow-screener-traffic" {
+  name = "allow_screener_traffic"
+  description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
+  vpc_id = "vpc-032e680f92b88bb68"
+
+  ingress {
+    description = "TCP traffic from VPC"
+    from_port = 80
+    to_port   = 80
+    protocol = "tcp"
+    cidr_blocks = ["0.0.0.0/8"]
+  }
+
+  egress {
+    description = "allow all outbound traffic from screener"
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+}
+
+resource "aws_ecr_repository" "eligibility-screener-repository" {
+  name                 = "eligibility-screener-repo"
+  image_tag_mutability = "MUTABLE"
+}
```

Comment:
note for a future ticket — you'll probably want to pull these ECR repository creations out into the account/bootstrap folder in the future so you can share docker images across environments (instead of having to rebuild images for each deployment) 

same with the ecs cluster below.

(this goes for both the eligibility screener and mock api!)